### PR TITLE
[HMA-4865] - Headline Card View

### PR DESCRIPTION
**Pull request by chrisbirch**
https://user-images.githubusercontent.com/3192947/140918825-f749a184-64d1-4cc4-98ae-de2e5b832833.mov
https://jira.tools.tax.service.gov.uk/browse/HMA-4865

### CODE REVIEW

File: SUICompanionApp/SUICompanionApp/Examples/ExampleImages.swift [hunk 1]
```diff
@@ -33,5 +33,8 @@ extension Example {
         var image: Image {
             return Image(self.rawValue)
         }
+        var uiImage: UIImage {
```
Comment (paulb00th) [inline SUICompanionApp/SUICompanionApp/Examples/ExampleImages.swift:36]: This doesn't seem to be used anywhere else in the PR

File: SUICompanionApp/SUICompanionApp/Organisms/ExampleOrganisms.swift [hunk 2]
```diff
@@ -61,3 +61,93 @@ extension Components.Organisms.PrimaryCardView: Examplable {
...
+                            print("How is this calcualted tapped")
+                        }.styled(.secondary(padding: 8, icon: Example.Images.info.image))
+                    }
+                )
+                Components.Organisms.HeadlineCardView(
+                    title: "Your PAYE income tax estimate",
+                    currencyAmount: 12345,
+                    content: {
+                        Text("This is the income tax we think you will have paid by the end of the year", style: .body)
+                        Text("TODO: This example is meant to be an inset view, see none swift ui version", style: .body)
```
Comment (paulb00th) [inline SUICompanionApp/SUICompanionApp/Organisms/ExampleOrganisms.swift:147]:
```suggestion
                        Text("TODO: This example is meant to be an inset view, see non-SwiftUI version", style: .body)
```

**Review by paulb00th — CHANGES_REQUESTED** [review]
Looks great - just a few nits from me

**Review by paulb00th — APPROVED** [review]

File: SUICompanionApp/SUICompanionApp/Organisms/ExampleOrganisms.swift [hunk 3]
```diff
@@ -61,3 +61,93 @@ extension Components.Organisms.PrimaryCardView: Examplable {
...
+                    content: {
+                        VStack(spacing: 0) {
+                            Text(Example.Text.longestIpsum, style: .body)
+                            Button(Example.Text.longIpsum) {
+                                print("Ipsum tapped")
+                            }.styled(.secondary(padding: 8))
+                        }
+                    }
+                )
+                Components.Organisms.HeadlineCardView(
```
Comment (tendai-moffatt) [inline SUICompanionApp/SUICompanionApp/Organisms/ExampleOrganisms.swift:132]: As discussed - lets lose this example until we make the [IconButtonView Component](https://jira.tools.tax.service.gov.uk/browse/HMA-4860)

File: Sources/SUIComponents/Atoms/Button+HMRC.swift [hunk 4]
```diff
@@ -18,7 +18,7 @@ import SwiftUI
 
 public enum HMRCButtonStyle {
     case primary
-    case secondary(padding: CGFloat = .spacer16)
+    case secondary(padding: CGFloat = .spacer16, icon: Image? = nil)
```
Comment (tendai-moffatt) [inline Sources/SUIComponents/Atoms/Button+HMRC.swift:21]: As discussed - lets lose this until we make the [IconButtonView Component](https://jira.tools.tax.service.gov.uk/browse/HMA-4860)

File: Sources/SUIComponents/Atoms/Button+HMRC.swift [hunk 5]
```diff
@@ -28,8 +28,8 @@ extension Button {
         switch style {
             case .primary:
                 self.buttonStyle(PrimaryButtonStyle())
-            case .secondary(let padding):
-                self.buttonStyle(SecondaryButtonStyle(padding))
+            case let .secondary(padding, image):
```
Comment (tendai-moffatt) [inline Sources/SUIComponents/Atoms/Button+HMRC.swift:31]: As discussed - lets lose this until we make the [IconButtonView Component](https://jira.tools.tax.service.gov.uk/browse/HMA-4860)

File: Sources/SUIComponents/Atoms/Button+HMRC.swift [hunk 6]
```diff
@@ -53,15 +53,20 @@ struct PrimaryButtonStyle: ButtonStyle {
 
 struct SecondaryButtonStyle: ButtonStyle {
     let padding: CGFloat
-    init(_ padding: CGFloat = .spacer16) {
+    let image: Image?
```
Comment (tendai-moffatt) [inline Sources/SUIComponents/Atoms/Button+HMRC.swift:56]: As discussed - lets lose this until we make the [IconButtonView Component](https://jira.tools.tax.service.gov.uk/browse/HMA-4860)

**Review by tendai-moffatt — CHANGES_REQUESTED** [review]

**Comment by tendai-moffatt** [comment]
Looks good apart from the [IconButtonView Component](https://jira.tools.tax.service.gov.uk/browse/HMA-4860) issue we discussed.

**Review by tendai-moffatt — APPROVED** [review]

**Pull request merged**